shaders/src/post_process.fs

```diff
@@ -9,5 +8,19 @@ void main() {
     inputs.color = clamp(inputs.color, -MEDIUMP_FLT_MAX, MEDIUMP_FLT_MAX);
```

Comment:
don't we need to remove this line?
Also I'm wondering if all the color outputs need to be clamped to +/-MEDIUMP_FLT_MAX on mobile?

Reply:
Ben already made that change, see the latest commit

Reply:
Yeah this caused a build issue on mobile. My fix is to clamp all of the color outputs. Is there ever a case where we would have a `highp` color output on mobile?

Reply:
I think we can cross that bridge later. We might indeed need a precision qualifier in theory.